core/models_template.py

```diff
@@ -0,0 +1,314 @@
+        db_table = 'dac1_current'
+
+
+class Dac2ACurrent(models.Model):
+    row_id = models.AutoField(primary_key=True)
+    recipient_code = models.SmallIntegerField()
+    recipient_name = models.TextField()
+    donor_code = models.SmallIntegerField()
+    donor_name = models.TextField()
+    part_code = models.IntegerField()
+    part_name = models.TextField()
+    aid_type_code = models.SmallIntegerField()
+    aid_type_name = models.TextField()
+    data_type = models.CharField(max_length=1)
+    amount_type = models.TextField()
+    time = models.SmallIntegerField()
+    year = models.SmallIntegerField()
+    value = models.DecimalField(max_digits=65535, decimal_places=65535, blank=True, null=True)
+    flags = models.TextField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'dac2a_current'
+
+
+class Dac2BCurrent(models.Model):
+    row_id = models.AutoField(primary_key=True)
+    recipient_code = models.SmallIntegerField()
+    recipient_name = models.TextField()
+    donor_code = models.SmallIntegerField()
+    donor_name = models.TextField()
+    part_code = models.IntegerField()
+    part_name = models.TextField()
+    aid_type_code = models.SmallIntegerField()
+    aid_type_name = models.TextField()
+    data_type = models.CharField(max_length=1)
+    amount_type = models.TextField()
+    time = models.SmallIntegerField()
+    year = models.SmallIntegerField()
+    value = models.DecimalField(max_digits=65535, decimal_places=65535, blank=True, null=True)
+    flags = models.TextField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'dac2b_current'
+
+
+class Dac5Current(models.Model):
+    row_id = models.AutoField(primary_key=True)
+    donor_code = models.SmallIntegerField()
+    donor_name = models.TextField()
+    sector_code = models.SmallIntegerField()
+    sector_name = models.TextField()
+    aid_type_code = models.IntegerField()
+    aid_type_name = models.TextField()
+    amount_type_code = models.CharField(max_length=1)
+    amount_type_name = models.TextField()
+    time = models.SmallIntegerField()
+    year = models.SmallIntegerField()
+    value = models.DecimalField(max_digits=65535, decimal_places=65535, blank=True, null=True)
+    flags = models.TextField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'dac5_current'
+
+
+class Operation(models.Model):
+    name = models.TextField()
+    description = models.TextField()
+    user = models.ForeignKey(User, models.DO_NOTHING)
+    operation_query = models.TextField()
+    theme = models.ForeignKey('Theme', models.DO_NOTHING)
+    sample_output_path = models.TextField(blank=True, null=True)
+    is_draft = models.BooleanField()
+    created_on = models.DateTimeField()
+    updated_on = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'operation'
+
+
+class OperationSteps(models.Model):
```

Comment:
I think all of these non-data models belong in `models.py`

Reply:
I'll just move everything to be managed by django and reduce on the complexity of our process